src/scripts/recon/shodan_favicon_hash/__main__.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+from base64 import encodebytes
+
+from mmh3 import hash
+from requests import get
+from requests.exceptions import RequestException
+
+from src.core.base.recon import ReconRunner, PossibleKeys
+from src.core.utils.response import ScriptResponse
+from src.core.utils.validators import validate_kwargs
+
+
+class Runner(ReconRunner):
+    def __init__(self, logger: str = __name__):
+        super(Runner, self).__init__(logger)
+
+    @validate_kwargs(PossibleKeys.KEYS)
+    def run(self, *args, **kwargs):
+        """
+        Returns hash of favicon.ico of given url.
+        :param args: args from core runner
+        :param kwargs: kwargs from core runner
+        :return: ScriptResponse message
+        """
+        url = kwargs.get("url")
+
+        try:
+            response = get(f"{url}/favicon.ico")
+        except RequestException as RE:
+            return ScriptResponse.error(
+                result=None,
+                message=f"Can't connect to {url}!\n"
+                        f"Error message: {RE}\n",
+            )
+
+        favicon = encodebytes(response.content)
+        favicon_hash = hash(favicon)
+
+        return ScriptResponse.success(
+            result=favicon_hash,
+            message=f"Successfully made favicon hash of {url}!\n"
```

Comment:
![ex](https://user-images.githubusercontent.com/25217914/87669863-e938ed80-c798-11ea-93a5-abe768c0d7e1.png)

Don't know why, but it inserts unintended line breaks.

Reply:
That's okay, try to use `print` instead of `pprint` :)